lesson01/assignment1/ES6Methods.js

```diff
@@ -0,0 +1,86 @@
+
+//Setup JQuery function
+$(function () {
+  //Create variables and array
+        const input = $('.input'),
+              submit = $('.submit'),
+              resultSum = $('.sum'),
+              findInArray =[12,14,16,18,36,72,216];
+//Link javascript to submit button in html
+submit.on('click', function () {
+        //Take the input value and parse it into a number
+        const val = input.val();
+        const num = parseInt(val);
+        //Varify that the user input is a number
+        if (!isNaN(num)) {
+        //See if number input is greater than any in array
+        const sum = findInArray.filter(value => value > num);
+        //Show result on the page
+        resultSum.text(sum);
+        //Reset the input value to nothing
+        input.val('');
+      }
+   });
+});
+
+//Setup JQuery function
+$(function () {
+  //Create variables
+        const input = $('.inputtwo'),
+              submit = $('.submittwo'),
+              resultSum = $('.sumtwo'),
+              word = "magic";
+//Link javascript to submit button in html
+submit.on('click', function () {
+        //Pickup input value
+        val = input.val();
+        //See if input matches anything in keyword
+        const searchword = word.includes(val);
+        resultSum.text(searchword);
+        //Reset the input value to nothing
+        input.val('');
+   });
+});
```

Comment:
This works ok on a string ('magic'), but there's also an `includes` function for arrays, which might make more sense for something like this. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes Right now, if a user enters 'gi' in the form, it will succeed. If, instead, the value of `word` was an array like `["magic", "unicorn", "mage"]`, then 'gi' wouldn't match, but a full word like 'mage' would. Your code is just fine, this is a matter of user experience 😄